code/game/objects/items/toys.dm

```diff
@@ -1091,4 +1091,29 @@ obj/item/toy/cards/deck/syndicate
 	desc = "An adorable stuffed toy that resembles a space carp."
 	icon = 'icons/obj/toy.dmi'
 	icon_state = "carpplushie"
-	w_class = 2.0
\ No newline at end of file
+	w_class = 2.0
+
+/*
+ * Toy big red button
+ */
+/obj/item/toy/redbutton
+	name = "\improper Big Red Button toy"

```

Comment:
Big Red Button may be capitalised to empathise its iconic meaning as it can become pretty much personal name, like Big Bad Wolf